bgpd/bgp_vty.c

```diff
@@ -1860,6 +1878,31 @@ DEFUN (no_bgp_always_compare_med,
 	return CMD_SUCCESS;
 }
 
+
+DEFUN(bgp_ebgp_requires_route_map, bgp_ebgp_requires_route_map_cmd,
+      "bgp ebgp-requires-route-map",
+      "BGP specific commands\n"
+      "Require in and out route-maps for eBGP peers (RFC8212)\n")
+{
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+	bgp->ebgp_requires_route_map = DEFAULT_EBGP_POLICY_ENABLED;
+	bgp_clear_all_ebgp_peers(bgp);
```

Comment:
It doesn't make sense to clear all peers here, for two reasons:

* any peer that does have in- & outbound route-maps is not affected, so we're causing a session drop for no reason.
* if we have soft-reconfiguration inbound or route-refresh enabled, we can soft-clear to apply the setting.

Reply:
The initial version was without clearing the session. I will update it.